capstone/src/main/resources/static/event.js

```diff
@@ -1,25 +1,72 @@
+/* Function to prefill event information if editing event */
+function loadEventInfo() {
+  const event = window.location.hash.substring(1);
```

Comment:
I left the same comment on Jenny's PR -- it is unusual to use the anchor tag (`#section`) to convey variables' values. It's more typical to use the query string `?key=value&key2=value2` for this.
When using the query string, you can use URLSearchParams to extract individual values.

https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams